functional-test/src/test/groovy/org/rundeck/tests/functional/selenium/jobs/ScheduledJobSpec.groovy

```diff
@@ -30,7 +32,7 @@ class ScheduledJobSpec extends SeleniumBase{
         loginPage.login(TEST_USER, TEST_PASS)
         jobListPage.go()
         jobListPage.validatePage()
-        hold(15)
+        waitFor(ExecutionUtils.Retrievers.executionsForProject(client, projectName), {it.size() >= 1}, WaitingTime.EXCESSIVE )
```

Comment:
I'm wondering what would it take to simplify this call even further. Something like:
```
waitFor ExecutionUtils.Retrievers.executionsForProject(client, projectName)
```

Here is the proposal:
https://github.com/rundeck/rundeck/pull/9369/files#diff-985b774f9b65ca78f876471f1100d1f1436ada846f7e811910150d9c0a6a2650R25

It would `accept` a non-empty list only.
What do you think @smartinellibenedetti ?